app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -116,57 +54,108 @@ def test_check_with_empty_queue_pushes_nothing(self):
         tasks.nimbus_check_kinto_push_queue_by_application(
             NimbusExperiment.Application.DESKTOP
         )
+        self.mock_kinto_client.patch_collection.assert_not_called()
         self.mock_push_task.assert_not_called()
+        self.mock_pause_task.assert_not_called()
         self.mock_end_task.assert_not_called()
 
-    def test_check_experiment_with_no_approved_publish_status_pushes_nothing(self):
-        for publish_status in [
-            NimbusExperiment.PublishStatus.IDLE,
-            NimbusExperiment.PublishStatus.REVIEW,
-            NimbusExperiment.PublishStatus.WAITING,
-        ]:
-            NimbusExperimentFactory.create(
-                publish_status=publish_status,
-                application=NimbusExperiment.Application.DESKTOP,
-            )
+    def test_check_with_no_approved_publish_status_pushes_nothing(self):
```

Comment:
Yeesh, github really doesn't know what to do with these changes in a diff does it? 🤷‍♂️